stock_batch_picking/i18n/de.po

```diff
@@ -0,0 +1,396 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# * stock_batch_picking
+# 
+# Translators:
+# Rudolf Schnapka <rs@techno-flex.de>, 2017
+# OCA Transbot <transbot@odoo-community.org>, 2017
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 9.0c\n"
```

Comment:
Odoo server version should be changed no ?

Reply:
Well, that's informative, and as the same translation file serves for all versions from 9 to 11, it's not needed to change it.